modules/transforms_update_meta_data.ipynb

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
<p>Could you please help also add the install logic like other tutorial notebooks?</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/758/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>